core/src/surplus.ts

```diff
@@ -54,11 +54,22 @@ const getAuctionState = async (network: string, earliestEndDate: Date, greatestB
     return 'just-started';
 };
 
+const _getContractAuctionDuration = async (contract: Contract) => {
+    return await contract.tau();
+};
+
+const getContractAuctionDuration = memoizee(_getContractAuctionDuration, {
+    maxAge: 24 * 60 * 60 * 1000,
+    promise: true,
+    length: 2,
```

Comment:
This indicates number of arguments it needs to cache
```suggestion
    length: 1,
```